src/runtime/static.ts

```diff
@@ -4,6 +4,7 @@ import {
   getAsset,
   readAsset,
   isPublicAssetURL,
+  PublicAsset,
```

Comment:
```suggestion
 type PublicAsset,
```
Should this one be a type import ?

Reply:
Yes 👍